### PR TITLE
fix: add the untranslated text content of Chart page, Sql Editor page and some dashboard label in messages.pot

### PR DESCRIPTION
**Pull request by cdmikechen**
### SUMMARY
Add the untranslated text content in Chart page, Sql Editor page and some dashboard label in messages.pot

### TEST PLAN

It can be viewed in the browser
chartlist2
auto_complete

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/dashboard/components/Header.jsx [hunk 1]
```diff
@@ -482,7 +482,7 @@ class Header extends React.PureComponent {
             <>
               <span
                 role="button"
-                title="Edit dashboard"
+                title={t('Edit Dashboard')}
```
Comment (ktmud) [inline superset-frontend/src/dashboard/components/Header.jsx:485]: We use sentence case for most UI elements: https://github.com/apache/superset/blob/master/CONTRIBUTING.md#capitalization-guidelines

**Review by ktmud — APPROVED** [review]
LGTM except one place on casing

**Review by zhaoyongjie — APPROVED** [review]
LGTM

File: superset-frontend/src/dashboard/components/Header.jsx (hunk 1 above)
Comment (cdmikechen) [reply superset-frontend/src/dashboard/components/Header.jsx]: @ktmud OK~
I've changed `Edit Dashboard` to `Edit dashboard`

**Comment by junlincc** [comment]
@cdmikechen mind fixing the conflicts real quick? ready to go~

**Review by villebro — APPROVED** [review]

**Comment by cdmikechen** [comment]
@junlincc  Hi~ 
The conflict has been resolved.

**Pull request merged**